src/main/scala/org/broadinstitute/dsde/workbench/sam/api/UserRoutes.scala

```diff
@@ -39,15 +40,24 @@ trait UserRoutes extends UserInfoDirectives with SamRequestContextDirectives {
       (pathPrefix("v1") | pathEndOrSingleSlash) {
         pathEndOrSingleSlash {
           post {
-            withTermsOfServiceAcceptance {
+            entity(as[TermsOfServiceAcceptance]) { tos =>
```

Comment:
I'm curious- why break this line out of the `withTermsOfServiceAcceptance` directive if it's ultimately going to be passed into the directive?